### DISCUSSION TITLE
bug: flopy.plot.PlotCrossSection not plotting projected coords

### DISCUSSION THREAD

**Discussion by kerrybardot** [modflowpy/flopy#2290] [Q&A] [answered]
flopy.plot.PlotCrossSection(model=model, line={"line": [(X0, Y0),(X1, Y1)]}) plots with orthogonal coordinates instead of projected.
Just wondering if projected coords could be used?

For example, is model domain is 1000 x 1000m, and line = [(0,0),(1000, 1000)] is plotted, then the line is (1000^2 + 1000^2)^0.5 long.

Maybe there is a reason you don't project it, but its making my plots a bit deceiving : )

**Comment by jlarsen-usgs** [comment, marked as answer]
@kerrybardot 

I'm going to move this to discussions. To plot cross sections using geographic/projected coordinates use:
```python
xc = flopy.plot.PlotCrossSection(model=model, line={"line":[(x0, y0), (x1, y1)]}, geographic_coords=True)
```
